.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,55 @@
+name: Nightly Fuzz
+
+on:
+  push:
+   schedule:
+     - cron: "0 0 * * *" # everyday
+
+permissions:
+  pull-requests: write
+
+jobs:
+  fuzz:
+    name: Fuzz
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: maven
+      - name: Setup `wasmtime`
+        uses: bytecodealliance/actions/wasmtime/setup@v1
+        with:
+          version: "v17.0.0"
+      - name: Setup `wasm-tools`
+        uses: bytecodealliance/actions/wasm-tools/setup@v1
+        with:
+          version: "1.0.57"
+      - name: Fuzz Test
+        # TODO: change before merging!
+        if: github.repository_owner == 'andreaTP'
```

Comment:
if you want to try it on your personal fork, you need to provide a `GH_TOKEN` and change this line.